cirq-google/cirq_google/devices/known_devices.py

```diff
@@ -175,7 +175,7 @@ def create_device_proto_for_qubits(
                 gate = gs_proto.valid_gates.add()
                 gate.id = gate_id
 
-                if not isinstance(serializer, op_serializer.GateOpSerializer):
+                if not isinstance(serializer, op_serializer._GateOpSerializer):
```

Comment:
This needs to handle both GateOpSerializer and _GateOpSerilalzer, right?

Reply:
`_GateOpSerializer` is a superclass of `GateOpSerializer` so both are handled